test/integration/verify/controls/azure_security_center_policies.rb

```diff
@@ -6,7 +6,7 @@
     its('policy_names') { should include('default', resource_group) }
   end
 
-  describe azurerm_security_center_policies.where(name: resource_group) do
+  describe azurerm_security_center_policies.where('name' => resource_group) do
```

Comment:
Why change this?

Do we want one use of the where with a string to make sure it warns and converts? The test should be separate and have a name to say that's the thing we're testing. The above would look like a mistake in the test code to my eyes, possibly evidence that keys _should_ be strings, to others.

Reply:
That's left over from my manual test. I need to remove that and replace it with a unit test.